OutOfSchool/OutOfSchool.WebApi/Services/WorkshopServicesCombiner.cs

```diff
@@ -49,11 +57,15 @@ public async Task<WorkshopDTO> Update(WorkshopDTO dto)
         {
             var workshop = await databaseService.Update(dto).ConfigureAwait(false);
 
-            var esResultIsValid = await elasticsearchService.Update(workshop.ToESModel()).ConfigureAwait(false);
+            //var esResultIsValid = await elasticsearchService.Update(workshop.ToESModel()).ConfigureAwait(false);
+
+            var esResultIsValid = false;
 
             if (!esResultIsValid)
             {
```

Comment:
probably this code should be swept

Reply:
it is only for test reason. when I will be ready for merge I will change it.

Reply:
Pls, write `//TODO` comments for such cases, it will be helpful both for you (just Ctrl-F all TODOs and nothing is missed) and for reviewers to understand you intention

Reply:
thank you. I will.